entity-service-impl/src/test/java/org/hypertrace/entity/query/service/EntityAttributeMappingTest.java

```diff
@@ -73,4 +76,41 @@ void returnsEmptyIfNoMapping() {
         Optional.empty(),
         attributeMapping.getDocStorePathByAttributeId(mockRequestContext, "some-id"));
   }
+
+  @Test
+  void testIsMultiValueAttribute() {
+    when(mockRequestContext.call(any())).thenCallRealMethod();
+    EntityAttributeMapping attributeMapping =
+        new EntityAttributeMapping(this.mockAttributeClient, Collections.emptyMap());
+    AttributeMetadata singleValueAttributeData =
+        AttributeMetadata.newBuilder()
+            .setKey("some-key")
+            .setValueKind(AttributeKind.TYPE_STRING)
+            .addSources(AttributeSource.EDS)
+            .build();
+    when(this.mockAttributeClient.get("some-id")).thenReturn(Single.just(singleValueAttributeData));
+
+    // Empty result, since the mock metadata doesn't have an entity source
```

Comment:
this comment doesn't match the input (it has an entity source, but the wrong value kind)

Reply:
My bad, removed it